.github/workflows/ci.yml

```
@@ -85,17 +85,33 @@ jobs:
       run: echo "::set-output name=version::$(cmake --version | head -n1 | awk '{print $3}')"
       shell: bash
 
+    - name: Runner-info
+      id: runner-info
+      run: |
+        env
+        echo "::set-output name=Info::$ImageOS-$ImageVersion"
```

`Env` context works only for variables directly defined in the workflow. Therefore we need to use an extra step to be able to read the values of environment variables that are not defined directly in the workflow.

---

For consistency with the rest of the workflow, I'd suggest using a lowercase variable name.
```suggestion
        echo "::set-output name=info::$ImageOS-$ImageVersion"
```